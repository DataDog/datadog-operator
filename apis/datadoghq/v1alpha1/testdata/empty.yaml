apiVersion: datadoghq.com/v1alpha1
kind: DatadogAgent
metadata:
  name: datadog