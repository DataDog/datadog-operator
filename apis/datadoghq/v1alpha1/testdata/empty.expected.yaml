metadata:
  creationTimestamp: null
  name: datadog
spec:
  features: {}
status: {}
